code/game/machinery/doors/windowdoor.dm

```diff
@@ -315,3 +324,36 @@
 	dir = SOUTH
 	icon_state = "rightsecure"
 	base_state = "rightsecure"
+
+
+/obj/machinery/door/window/briefing/northleft
+	dir = NORTH
+
+/obj/machinery/door/window/briefing/eastleft
+	dir = EAST
+
+/obj/machinery/door/window/briefing/westleft
+	dir = WEST
+
+/obj/machinery/door/window/briefing/southleft
+	dir = SOUTH
+
+/obj/machinery/door/window/briefing/northright
+	dir = NORTH
+	icon_state = "rightsecure"
+	base_state = "rightsecure"
+
+/obj/machinery/door/window/briefing/eastright
+	dir = EAST
+	icon_state = "rightsecure"
+	base_state = "rightsecure"
+
+/obj/machinery/door/window/briefing/westright
+	dir = WEST
```

Comment:
very no

Reply:
This can be removed, it's not a must, if it makes the mappers' lives harder.